lib/ansible/modules/cloud/vmware/vmware_host_vmhba_info.py

```diff
@@ -167,11 +172,19 @@ def gather_host_vmhba_info(self):
                     hba_info['model'] = hba.model
                     hba_info['driver'] = hba.driver
                     try:
-                        hba_info['node_wwn'] = self.format_number(hba.nodeWorldWideName)
+                        if isinstance(hba, vim.host.FibreChannelHba) or \
```

Comment:
```suggestion
                        if isinstance(hba, (vim.host.FibreChannelHba, vim.host.FibreChannelOverEthernetHba)):
```

isinstance takes a tuple as well :)

Reply:
Sorry, `FibreChannelOverEthernetHba` is mistake.
Because the property of `nodeWorldWideName` and `portWorldWideName` did not exist in `FibreChannelOverEthernetHba`.
https://vdc-download.vmware.com/vmwb-repository/dcr-public/6b586ed2-655c-49d9-9029-bc416323cb22/fa0b429a-a695-4c11-b7d2-2cbc284049dc/doc/vim.host.FibreChannelOverEthernetHba.html

Reply:
Thank you for the advice!
`FibreChannelOverEthernetHba` inherited the properties of `FibreChannelHba`, so I modified it according to the advice.
https://vdc-download.vmware.com/vmwb-repository/dcr-public/6b586ed2-655c-49d9-9029-bc416323cb22/fa0b429a-a695-4c11-b7d2-2cbc284049dc/doc/vim.host.FibreChannelOverEthernetHba.html